chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -494,23 +494,49 @@ mcCombinations.prototype.possibleCombinations = [
 		maxfields: 1,
 		score: 0,
 		autoSubmit: false
+	},
+	{
+		combinationId: 'form001',
```

Comment:
The present algorithm won't find any combination for fields such as first name and last name on vk.com, but it is dependent on found combinations to fill in the fields.